x-pack/plugins/alerting/server/alerts_client/alerts_client.ts

```diff
@@ -402,6 +412,33 @@ export class AlertsClient<
         );
       }
     }
+
+    function getBulkMeta(
```

Comment:
Here are the main changes to how we "write" documents, depending on create/update.  Updates are via `index` with OCC fields applied.  Creating new docs is via `create`.  This is the way (of data streams).

Should note here, this is a behavior change.  Previously, we just did `index`, which for alias/indices just overwrote the existing document.  So in race conditions, the second one won.  Now, with OCC, the second one will lose.  I figure, doesn't matter which loses.  Though we may see some specific patterns where this behavior change **DOES** matter.  

We need to fix this problem somehow, and the PR points to a specific GH issue surrounding this.